src/Utilities.mss

```diff
@@ -462,3 +474,75 @@ function GetNongraceParentBeam (noteRest, layer) {
     return null;
 }  //$end
 
+function AppendToLayer (meielement, l, beam, tuplet) {
```

Comment:
The motivation for this function is to properly append everything that can be inside a beam or tuplet.  However it can be used to append anything to a layer.

The elements it's currently used for are `<note>`s, `<rest>`s, `<chord>`s, `<space>`s and `<clef>`s.  It should be used for `<fTrem>`s and `<bTrem>`s as well when support for them is added.

As far as I can see, the only elements that do not rely on this function are `<mRest>`, `<mRpt>` and `<mRpt2>`.  They don't need the tuplet and beam checking overhead but *could* no problem be appended using this function.  For consistency's sake, that might be worth a consideration.